.travis.yml

```diff
@@ -7,11 +7,10 @@ script: bundle exec rake travis
 matrix:
   fast_finish: true
   include:
-    - rvm: 1.9.3
```

Comment:
We didn't decide about `1.9.3` yet. Perhaps the end of support might change this scenario.

@tokengeek @geemus Thoughts?